node/internal_binding/constants.ts

```diff
@@ -1,145 +1,633 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
Manually obtained the constants for all 3 os, this one is the right one. However the error codes won't match those in UV (they are not supposed to)